plugin-server/src/main/services/http-server.ts

```diff
@@ -1,38 +1,47 @@
+import * as Sentry from '@sentry/node'
 import { createServer, IncomingMessage, Server, ServerResponse } from 'http'
 
-import { healthcheck } from '../../healthcheck'
 import { status } from '../../utils/status'
-import { stalenessCheck } from '../../utils/utils'
```

Comment:
same & you're sure we don't want this anymore, i.e. we could have problems with staleness outside of Kafka too, no?

Reply:
Yeah but it's not currently being used actually